src/zui/ZUIOrganizeSidebar.tsx

```diff
@@ -56,7 +56,7 @@ const useStyles = makeStyles((theme) => ({
   toolbar: theme.mixins.toolbar,
 }));
 
-const OrganizeSidebar = (): JSX.Element => {
+const ZUIOrganizeSidebar = (): JSX.Element => {
```

Comment:
I'm not sure how I feel about this. No need to do anything about this now, but I don't think this is a ZUI component. It's not a component that will be used in more than one place (maybe more than one call site in the code, but only one visual use case).

Let's come back to it when we structure ZUI once it's bigger.

Reply:
Definitely! We begin here, and keep adjusting until it works for us :)